recipes/libpq/all/conanfile.py

```diff
@@ -46,15 +40,21 @@ def config_options(self):
     def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
-        if self.settings.compiler != "Visual Studio" and self.settings.os == "Windows":
-            if self.options.shared:
-                raise ConanInvalidConfiguration("static mingw build is not possible")
+        if self.settings.compiler != "Visual Studio" and self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("shared mingw build is not possible")
```

Comment:
I'm looking at this now, and shared mingw@Windows are possible

Reply:
Would be nice. I saw also that build of libpq did not fail. It installs dll file, static libs, but not import lib...